tests/Operator/SecondQuantized/USQHamiltonian_test.cpp

```diff
@@ -0,0 +1,85 @@
+// This file is part of GQCG-gqcp.
+// 
+// Copyright (C) 2017-2019  the GQCG developers
+// 
+// GQCG-gqcp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// GQCG-gqcp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-gqcp.  If not, see <http://www.gnu.org/licenses/>.
+// 
+#define BOOST_TEST_MODULE "USQHamiltonian"
+
+#include <boost/test/unit_test.hpp>
+
+#include "Operator/SecondQuantized/USQHamiltonian.hpp"
+#include "RHF/PlainRHFSCFSolver.hpp"
+#include "Utilities/miscellaneous.hpp"
+#include "Utilities/linalg.hpp"
+
+#include <boost/math/constants/constants.hpp>
+
+
+
```

Comment:
Could you add a clear test description, in the same format that we add method descriptions? I.e. with `/** * */`.